integration-test/pom.xml

```diff
@@ -167,31 +167,25 @@
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_${scala.binary.version}</artifactId>
+      <scope>test</scope>
```

Comment:
Oops you are right. Those ITs are not supposed to be under `test/`.
There's a pending work item to generate a livy-intergration test jar to run Livy IT without Maven.